modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -85,8 +85,58 @@ Note: If the payload is used as part of the message log, do not configure the Li
 |After Calling API |Set to cause your logging policy to log messages after the API endpoint is called, taking into account the order in which this policy is applied |Optional
 |===
 
+== Example of Message Logging Policy Configured from the UI
 
-== Example of Logging Policy in XML
+To better understand how to use the Message Logging policy, consider the "Les Vetments" company that is in the business of retail clothing and more. To run their e-commerce portal successfully, Les Vetments needs to audit the status code of the requests made on their website.
+
+For example, if a customers buys or returns a product, or if the company adds or removes a product from the catalogue, the transactions must be logged. The IT manager at Les Vetments applies the Message Logging policy to their APIs to enable logging and viewing any data or transaction changes without making any modifications to the code.
+
+To log the attributes of the incoming and outgoing HTTP Message, Les Vetments applies the Message Logging policy to the API with the following UI configurations:
+
+* *Message*: #["Current HTTP Attributes are: " ++ attributes]
+
+The attributes in the payload must be logged.
+* *Conditional*: Blank
+
+Because Les Vetments requires all attributes, no Mule expression to filter messages is specified.
+* *Category*: Blank
+
+No prefix is required in the log sentence.
+* *Level*: Info
+
+All types of messages are to be logged: Info, Warn, Error, or Debug.
```

Comment:
Do we need to explain why or clarify further that specifying Info also includes Warn, Error, and Debug?

Reply:
Done! Thanks!